csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1516,6 +1728,22 @@ Valid values are:
     not_contacted
     open
 
+Each status is mutually exclusive - only one status is valid for a particular vulnerability at a particular time. As the vulnerability ages, a party’s involvement could move from state to state. However, in many cases, a document producer may choose not to issue CSAF documents at each state, or simply omit this element altogether. It is recommended, however, that vendors that issue CSAF documents indicating an open or in-progress Involvement should eventually expect to issue a document as `disputed` or `completed`.
+
+> The two vulnerability involvement status states, `contact_accepted` and `not_contacted` are intended for use by document producers other than vendors (such as research or coordinating entities).
+
+The value `completed` indicates that the vendor asserts that investigation of the vulnerability is complete. No additional information, fixes, or documentation from the vendor about the vulnerability should be expected to be released.
+
+The value `contact_accepted` indicates that the document producer attempted to contact the affected vendor.
+
+The value `disputed` indicates that the vendor disputes the vulnerability report in its entirety. Vendors should indicate this status when they believe that a vulnerability report regarding their product is completely inaccurate (that there is no real underlying security vulnerability) or that the technical issue being reported has no security implications.
+
+The value `in_progress` indicates that some hotfixes, permanent fixes, mitigations, workarounds, or patches may have been made available by the vendor, but more information or fixes may be released in the future. The use of this status by a vendor indicates that future information from the vendor about the vulnerability is to be expected.
```

Comment:
On first sight, value `in_progress` clashes with `product_status`->`under_investigation`. On second sight, I also see the use case of 3rd party organizations providing a CSAF document for a critical vulnerability and summarizing the state of the different vendors that are potentially affected. Is this the case? If yes, this use case should be explained.

Reply:
I understood the latter as the use case, where the reporting party may have no active role in the investigation. But, I may be wrong.

Reply:
This is now tracked as #221. The solution will be probably part of one of the next editor revisions.